selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java

```diff
@@ -69,8 +70,9 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    String newWorkspaceName = testWorkspace.getName() + NEW_WORKSPACE_SUFFIX;
-    workspaceServiceClient.delete(newWorkspaceName, user.getName());
+    if (factoryWsName != null) {
```

Comment:
when factoryWsName can be null?

Reply:
**factoryWsName** can be null when there is a problem with opening workspace created from factory.

Reply:
ok. when remove this if and fix the problem.

Reply:
The problem occurs at the Che side. It can't be fixed within the test.

Reply:
that problem? Do you have an issue for that?

Reply:
There is a [problem with switching from _Dashboard_ iframe to _Ide_ one](https://ci.codenvycorp.com/view/qa/job/nightly-selenium-tests/33/testReport/junit/org.eclipse.che.selenium.factory/CheckRunCommandFeatureTest/checkRunCommandFeatureTest/) sometimes happens. 
And you are right  - it's better to register an issue about it.